server/handler/fetcher.go

```diff
@@ -58,11 +58,11 @@ func (fc FetchedConfig) String() string {
 func (fc FetchedConfig) Description() string {
 	switch {
 	case fc.Missing():
-		return fmt.Sprintf("No policy found at ref=%s", fc.Ref)
+		return fmt.Sprintf("No policy found at %s", fc)
 	case fc.Invalid():
-		return fmt.Sprintf("Invalid configuration defined by ref=%s", fc.Ref)
+		return fmt.Sprintf("Invalid configuration defined by %s", fc)
 	}
-	return fmt.Sprintf("Valid policy found for ref=%s", fc.Ref)
+	return fmt.Sprintf("Valid policy found for %s", fc)
```

Comment:
The `String` method is more detailed than just a `ref`, and we seem to be a bit inconsistent in using the two formats. I think in all cases owner/repo/ref would be preferable to just ref, correct?

Reply:
Yeah, that sounds fine. I suspect the original use of only `ref` was acceptable because this always appears in the context of a log message or UI page that is already annotated with information about the repository.